MLIR.cmake

```diff
@@ -108,12 +108,15 @@ endfunction()
 #     Same semantics as target_include_directories().
 #   LINK_LIBS lib_targets...
 #     Same semantics as target_link_libraries().
+#   LINK_TARGET_ARCH_LIBS
```

Comment:
LLVM calls it LINK_COMPONENTS. If we want to keep the same nomenclature.

Reply:
ok let me update it